packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeEthInANutshellModal/InfoRow.tsx

```diff
@@ -0,0 +1,62 @@
+import { Badge, Paragraph, Row, useElevation } from '@trezor/components';
+import { Elevation, mapElevationToBorder, spacings, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+
+const ListItem = styled.li<{ $elevation: Elevation }>`
+    position: relative;
+    padding-left: ${spacingsPx.xl};
+    margin-bottom: ${spacingsPx.md};
+    list-style-type: none;
+
+    &::before {
+        position: absolute;
+        content: '';
+        left: 0;
+        top: 50%;
+        transform: translateY(-50%);
+        width: ${spacingsPx.md};
+        height: ${spacingsPx.md};
+        border-radius: 50%;
+        border: ${spacingsPx.xxs} solid ${mapElevationToBorder};
+    }
+
+    &:not(:last-child)::after {
+        position: absolute;
+        content: '';
+        left: calc(${spacingsPx.md} / 2 - ${spacingsPx.xxxs} / 2);
```

Comment:
You don't need to use calc, use spacings (without Px) instead, like this `left: ${spacings.md / 2 ...}px`. Also, why use `spacingsPx.md / 2` and not `spacingsPx.xs`?

Reply:
done, https://github.com/trezor/trezor-suite/pull/14410/commits/3dbc96661afb16f226d41fa245ff8d0f7c840850